script.js

```diff
@@ -1,46 +1,61 @@
 function minimize(section) {
     let text = document.getElementById(section.toString());
-    if (text.style.display == "none") {
-        text.style.display = "";
-    } else {
-        text.style.display = "none";
-    }
+    text.style.display = text.style.display === 'none' ?
+        '' :
+        'none'
 }
 
 function rotate(header) {
-    let children = document.getElementById(header).childNodes;
-    for (let i = 0; i < children.length; i += 1) {
-        let cur = children[i];
-        if (cur.innerHTML == '❯') {
-            if (cur.className == 'rotate') {
-                cur.className = 'arrow';
-            } else {
-                cur.className = 'rotate';
-            }
+    for (const child of document.getElementById(header).childNodes) {
+        if (child.innerHTML == '❯') {
+            child.className = child.className === 'rotate' ?
+                'arrow' :
+                'rotate'
         }
     }
 }
-var pText = [];
-pText[0] = "<em>Java</em> Players attempt to avoid pathfinding enemies in a pseudorandomly generating world, or play in a custom-built world.";
-pText[1] = "<em>Java</em> Users interact with a dynamic map of the city of Berkeley and find the shortest path between point A and point B.";
-pText[2] = "<em>Python</em> In a parody of <em>Plants Vs. Zombies</em>, players attempt to prevent the Bees from reaching the Ant Queen with a number of tactical Ants.";
-pText[3] = "<em>Java</em> Users can enter a sequence of characters and get a nice melody outputted using the Karplus-Strong algorithm to convert it to \"plucking\".";
-pText[4] = "<em>Python</em> This was a group project to see whether a training an algorithm on short-term trends and financial reports could outperform the market."
-var pTitles = [];
-pTitles[0] = "Build Your Own World";
-pTitles[1] = "BearMaps";
-pTitles[2] = "Ants vs. Bees";
-pTitles[3] = "Digital Rockstar";
-pTitles[4] = "Beating the Market";
+
+const pText = [
+    "<em>Java</em> Players attempt to avoid pathfinding enemies in a pseudorandomly generating world, or play in a custom-built world.",
+    "<em>Java</em> Users interact with a dynamic map of the city of Berkeley and find the shortest path between point A and point B.",
+    "<em>Python</em> In a parody of <em>Plants Vs. Zombies</em>, players attempt to prevent the Bees from reaching the Ant Queen with a number of tactical Ants.",
+    "<em>Java</em> Users can enter a sequence of characters and get a nice melody outputted using the Karplus-Strong algorithm to convert it to \"plucking\".",
+    "<em>Python</em> This was a group project to see whether a training an algorithm on short-term trends and financial reports could outperform the market."
+]
+
+const pTitles = [
+    "Build Your Own World",
+    "BearMaps",
+    "Ants vs. Bees",
+    "Digital Rockstar",
+    "Beating the Market"
+]
```

Comment:
JS lets you define arrays using an [array literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#literals); incidentally, arrays can be heterogeneous--they can contain members of different types, like `[1, "234"]`.

Reply:
`const a = ...` means "I will never reassign `a` to another value". It's not the same as `final` in Java, since you can still modify the array itself.